src/m365/planner/commands/task/task-get.ts

```diff
@@ -31,25 +31,42 @@ class PlannerTaskGetCommand extends GraphCommand {
   }
 
   public get description(): string {
-    return 'Retrieve the the specified planner task';
+    return 'Retrieve the specified planner task';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.title = typeof args.options.title !== 'undefined';
+    telemetryProps.bucketId = typeof args.options.bucketId !== 'undefined';
+    telemetryProps.bucketName = typeof args.options.bucketName !== 'undefined';
+    telemetryProps.planId = typeof args.options.planId !== 'undefined';
+    telemetryProps.planName = typeof args.options.planName !== 'undefined';
+    telemetryProps.ownerGroupId = typeof args.options.ownerGroupId !== 'undefined';
+    telemetryProps.ownerGroupName = typeof args.options.ownerGroupName !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'title', 'priority'];
```

Comment:
As we're returning a single item, we don't need to specify defaultProperties. DefaultProperties are meant for use with `list` commands that return multiple items so that we can format them in a readable way in text output.